contracts/GenArt721CoreV3.sol

```diff
@@ -963,7 +1025,31 @@ contract GenArt721CoreV3 is
         external
         onlyUnlocked(_projectId)
         onlyArtistOrAdminACL(_projectId, this.updateProjectAspectRatio.selector)
+        onlyNonEmptyString(_aspectRatio)
     {
+        // Perform more detailed input validation for aspect ratio.
```

Comment:
nit: I think this allows the string `.` - we should require at least one number

Reply:
Good call–updating validation logic and tests.